### PR TITLE
LUN-32: Setup Release Notes

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: tools/parse-release-notes [hunk 1]
```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node
```
Comment (SteffiStuffel) [inline tools/parse-release-notes:1]: Almost the same as for integreat. Just removes web-option and language-selection, as lunes is just available in german.

**Comment by SteffiStuffel** [comment]
Have not tested this yet. I guess I can't without deploying?

File: .circleci/src/jobs/move_release_notes.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+resource_class: small
+steps:
+  - checkout
+  - prepare_workspace
+  - restore_environment_variables
+  - restore_yarn_tools_cache
+  - run:
+      name: Move release notes
+      command: tools/move-release-notes move-to ${NEW_VERSION_NAME} --deliverino-private-key ${DELIVERINO_PRIVATE_KEY} --owner ${CIRCLE_PROJECT_USERNAME} --repo ${CIRCLE_PROJECT_REPONAME} --branch ${CIRCLE_BRANCH}
+  #- notify TDB in LUN-30
```
Comment (klinzo) [inline .circleci/src/jobs/move_release_notes.yml:13]: With the following you can view all todos in intellij directly
```suggestion
  #- notify TODO in LUN-30
```

File: .circleci/src/workflows/prod_delivery.yml [hunk 3]
```diff
@@ -15,7 +15,15 @@ jobs:
       production_delivery: true # TODO test in mid of august
       requires:
         - build_android
-#  - deliver_ios: # LUN-29
-#      production_delivery: true
-#      requires:
-#        - build_ios
+  #  - deliver_ios: # LUN-29
```
Comment (klinzo) [inline .circleci/src/workflows/prod_delivery.yml:18]: Same as well, would add TODOs

**Review by klinzo — COMMENTED** [review]
Did not review the scripts. Seems to be fine. If you want to test this you can just remove the current circleci config and just execute the steps you want to test. Best to do this in a separate PR or force push afterwards tough.

**Review by klinzo — APPROVED** [review]

**Pull request merged**